docs/03-deploy.Rmd

```diff
@@ -12,7 +12,7 @@ As we just mentioned, Netlify\index{Netlify} allows you to quickly publish a web
 
 Basically, you have to host all source files of your website in a GIT repository.^[If the contents of your `blogdown` site are not in the root directory of your GIT repository, Netlify will not build.] You do not need to put the `public/` directory under version control^[You can add `public` to `.gitignore` to ignore it in GIT.] because it will be automatically generated. Currently, Netlify supports GIT repositories hosted on GitHub, GitLab, and BitBucket. With any of these accounts, you can log into Netlify from its homepage and follow the guide to create a new site from your GIT repository.
 
-Netlify supports several static website generators, including Jekyll and Hugo. For a new site, you have to specify a command to build your website, as well as the path of the publish directory. Netlify also supports multiple versions of Hugo, so the build command can be the default `hugo`. The default version is 0.17, which is too old, and we recommend that you use at least version 0.20. To specify a Hugo version greater or equal to 0.20, you need to create an environment variable `HUGO_VERSION` on Netlify. See the [Netlify documentation](https://www.netlify.com/docs/continuous-deployment/) for more information. The publish directory should be `public` unless you have changed it in your `config.toml`. Figure \@ref(fig:netlify-settings) shows the settings of the website https://t.yihui.org. You do not have to follow the exact settings for your own website; in particular, you may need to change the value of the environment variable `HUGO_VERSION` to a recent version of Hugo.^[By the time when this book is published, the version 0.24.1 may be too old.]
+Netlify supports several static website generators, including Jekyll and Hugo. For a new site, you have to specify a command to build your website, as well as the path of the publish directory. Netlify also supports multiple versions of Hugo, so the build command can be the default `hugo`. The default version is 0.17, which is too old, and we recommend that you use at least version 0.20. To specify a Hugo version greater or equal to 0.20, you need to create an environment variable `HUGO_VERSION` on Netlify. See the [Netlify documentation](https://www.netlify.com/docs/continuous-deployment/) for more information. The publish directory should be `public` unless you have changed it in your `config.toml`. Figure \@ref(fig:netlify-settings) shows the settings of the website https://t.yihui.org. You do not have to follow the exact settings for your own website; in particular, you may need to change the value of the environment variable `HUGO_VERSION` to a recent version of Hugo or add a [`netlify.toml`](https://docs.netlify.com/site-deploys/create-deploys/#template-configuration) to the root directory of your site and add the line `HUGO_VERSION = "0.79.1"`, for example.^[By the time when this book is published, the version 0.24.1 may be too old.]
```

Comment:
For the documentation of `netlify.toml`, should we link to https://docs.netlify.com/configure-builds/file-based-configuration/ instead?

We should definitely mention `blogdown::config_netlify()` here (available in the current dev version), which generates `netlify.toml` and sets the Hugo version automatically. The Netlify documentation is too general and probably also too technical to average users.

Reply:
Big :+1: I think I actually created `netlify.toml` in my website (setting it to 0.66.0) and then forgot causing mayhem (and hopefully a small but handy update to the great blogdown book)!

Reply:
Do you want me to make additional changes or are you happy to make post merge fixes (may be easier as you know exactly what `config_netlify()` is designed to do. May be worth mentioning how to update Hugo versions also. Context wowchemy and some other themes insist on recent versions so if anyone wants to migrate it will be useful know-how.

Reply:
Linking to #546

Reply:
I'll make the edit. Thanks!